runtime/dart_isolate.cc

```diff
@@ -652,6 +679,8 @@ bool DartIsolate::LoadLibraries() {
   DartIO::InitForIsolate(may_insecurely_connect_to_all_domains_,
                          domain_network_policy_);
 
+  InitDartFFIForIsolateGroup();
```

Comment:
This function will be called for every each newly created isolate and is therefore the wrong place.

I think this needs to go into `DartIsolate::CreateDartIsolateGroup` as we want to do this **once** for an isolate group (as native resolution & caching isn't per-isolate but per-isolate-group)

Reply:
Done.